src/LiveComponent/doc/index.rst

```diff
@@ -2302,6 +2302,131 @@ You can also trigger a specific "action" instead of a normal re-render:
         #}
     >
 
+Binding LiveProp to URL query parameters
+----------------------------------------
+
+.. versionadded:: 2.13
+
+    The ``url`` option was introduced in Live Components 2.13.
+
+You can bind your component LiveProp's values to URL query parameters thanks to the ``url`` option::
+
+    // src/Components/SearchModule.php
```

Comment:
```
    // src/Twig/Components/SearchModule.php
```

right?

Reply:
Other examples in the doc live in `src/Components`, not sure why this one should be different?